utils/config.go

```diff
@@ -5,54 +5,53 @@ import (
 	"log"
 	"os"
 
-	"github.com/denverdino/aliyungo/acm"
+	"github.com/fredliang44/aliyungo/acm"
 	"github.com/gin-gonic/gin"
 	yaml "gopkg.in/yaml.v2"
 )
 
 // Config is a struct for backend config
 type Config struct {
 	APPName string `default:"Gin App"`
-
+	ShortUrl string `default:"uniqs.cc"`
 	Server struct {
-		Host      string `default:"127.0.0.1"`
-		Hostname  string `default:"open.hustunique.com"`
-		Port      string `default:"9012"`
-		SecretKey string `default:"SecretKey"`
+		Host      string `default:"127.0.0.1";yaml:"host"`
```

Comment:
The spliter of golang struct tag is ` ` (backspace), not `;`

Reference
https://medium.com/golangspec/tags-in-golang-3e5db0b8ef3e
https://stackoverflow.com/questions/18635671/how-to-define-multiple-name-tags-in-a-struct